tests/requirements.txt

```diff
@@ -0,0 +1,18 @@
+--extra-index-url https://packages.dea.ga.gov.au/
+setuptools
+datacube[performance,s3]==1.8.3
+aiobotocore[boto3,awscli]==1.1.1
+aiohttp==3.6.2
+pandas==1.0.5
```

Comment:
No need to pin pandas to older version anymore, let's test with the same setup we have in sandbox currently,

```
pandas==1.1.2
xarray==0.16.1
```